src/spiceypy/tests/test_wrapper.py

```diff
@@ -547,6 +548,12 @@ def test_ckcov():
     assert [
         [cover[i * 2], cover[i * 2 + 1]] for i in range(spice.wncard(cover))
     ] == expected_intervals
+    
+    
+def test_ckcov2():
+    ckid = spice.ckobj(ExtraKernels.v1jCk)[0]
+    cover = SPICEDOUBLE_CELL(200000)
```

Comment:
additional tests in `test_support_types.py` should be added to ensure test failures if the `__bool__` or `__len__` behavior changes, so that the current behavior (or whatever I work out to be correct later) can have visibility.